快速构建器/i18n/zh_CN.go

```diff
@@ -5,10 +5,10 @@ var I18nDict_zh_CN map[uint16]string = map[uint16]string{
 	ACME_FailedToSeek:                   "无效ACME文件，因为seek操作失败了。",
 	ACME_StructureErrorNotice:           "文件结构错误",
 	ACME_UnknownCommand:                 "未知ACME命令（文件错误）",
-	Auth_BackendError:                   "后端错误",
+	Auth_BackendError:                   "后端错误，别担心，这不是你的问题",
 	Auth_FailedToRequestEntry:           "未能请求租赁服入口，请检查租赁服等级设置是否关闭及租赁服密码是否正确。",
-	Auth_HelperNotCreated:               "辅助用户尚未创建，请前往用户中心进行创建。",
-	Auth_InvalidFBVersion:               "FastBuilder 版本无效，请更新。",
+	Auth_HelperNotCreated： 		    "辅助用户尚未创建，请前往用户中心进行创建。",
```

Comment:
```suggestion
	Auth_HelperNotCreated: 		    "辅助用户尚未创建，请前往用户中心进行创建。",
```
There is an invalid syntax.